skyportal/tests/api/test_sources.py

```diff
@@ -193,7 +193,7 @@ def test_token_user_post_new_source(upload_data_token, view_only_token, public_g
     npt.assert_almost_equal(data['data']['ra'], 234.22)
 
     saved_at = parser.parse(data['data']['groups'][0]['saved_at'] + " UTC")
-    assert abs(saved_at - t0) < timedelta(seconds=2)
+    assert abs(saved_at - t0) < timedelta(seconds=60)
```

Comment:
This failed locally with a difference of ~25 seconds once, even though the test completed within a few seconds...
This has also failed on CI